docs/user/check-ref/scalyr_wrapper.rst

```diff
@@ -33,6 +33,11 @@ For ``minutes`` and ``end`` you can also specify absolute times like "2017-10-11
     This method is used to retrieve the most common values for a field.
 
 
+.. py:method:: function(function, query, minutes=5, end=0)
```

Comment:
the function is called `function`?

Reply:
Do you mean the method-name or the parameter name? :smile: 
But yes, see https://github.com/zalando-zmon/zmon-worker/pull/435/files#diff-30aa66f82e2b759e3e0885123d52ca23R98

Reply:
wow very meta :)